Engine/ResourceManagement/Importer/Importer.cpp

```diff
@@ -54,10 +56,20 @@ bool Importer::ImportRequired(const Path& file_path)
 
 		Metafile* metafile = App->resources->metafile_manager->GetMetafile(*metafile_path);
 		assert(App->resources->metafile_manager->IsMetafileConsistent(*metafile));
-		return
-			metafile_path->GetModificationTimestamp() < file_path.GetModificationTimestamp()
-			|| metafile->version < Importer::IMPORTER_VERSION
-			|| !App->filesystem->Exists(metafile->exported_file_path);
+
+		if (metafile->version < Importer::IMPORTER_VERSION || !App->filesystem->Exists(metafile->exported_file_path))
+		{
+			return true;
+		}
+		else if (App->filesystem->Exists(metafile->exported_file_path))
```

Comment:
@yalania Given that you're calling this method twice here and on the if right above maybe it could be nice to declare a boolean right before these two blocks and reuse it

Reply:
Same